src/completion/list_raidset.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo -n `cat /proc/partitions | grep md | awk '{ print $4 }'`
```

Comment:
I've added this as a simple wrapper script, but because its a simple oneliner it would fit inside the XML . is it wise to make it a wrapper script or should i move it to a inline part of the XML?

Reply:
Maybe not, I don't think it has a potential to grow, and even if it does, it's easy to move to a separate script.